### PR TITLE
NOTIF-133 add clowdapp.yaml

### PR DESCRIPTION
**Pull request by lahavyuv86**

### CODE REVIEW

**Comment by bsquizz** [comment]
LGTM, should have no impact since this file isn't wired to an actual deployment yet.

File: deploy/clowdapp.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        readinessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /
+            port: 8000
+            scheme: HTTP
+          initialDelaySeconds: 35
+          periodSeconds: 5
+          successThreshold: 1
+          timeoutSeconds: 120
```
Comment (theute) [inline deploy/clowdapp.yaml:38]: Liveness and readiness probe seem wrong they don't match the current ones:
https://github.com/RedHatInsights/e2e-deploy/blob/master/templates/notifications/notifications-backend.yml

File: deploy/clowdapp.yaml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        env:
+        - name: CLOWDER_ENABLED
+          value: ${CLOWDER_ENABLED}
+        resources:
+          limits:
+            cpu: ${CPU_LIMIT}
+            memory: ${MEMORY_LIMIT}
+          requests:
```
Comment (theute) [inline deploy/clowdapp.yaml:46]: Can we also use parameters like it exists today:
https://github.com/RedHatInsights/e2e-deploy/blob/56d3c6990c7ba1250c536ed691cea45b140f65e3/templates/notifications/notifications-backend.yml#L135-L137

File: deploy/clowdapp.yaml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  required: true
+- description: Image name
+  name: IMAGE
+  value: quay.io/cloudservices/notifications-backend
+- description: Determines Clowder deployment
+  name: CLOWDER_ENABLED
+  value: "false"
+- description: ClowdEnv Name
+  name: ENV_NAME
+  requred: false
```
Comment (theute) [inline deploy/clowdapp.yaml:85]: `requred` ;-)

**Review by theute — CHANGES_REQUESTED** [review]
I can't really tell if it's correct or not, but just compared with the existing and noticed differences (and a typo)

File: deploy/clowdapp.yaml (hunk 1 above)
Comment (lahavyuv86) [reply deploy/clowdapp.yaml]: Fixed

File: deploy/clowdapp.yaml (hunk 2 above)
Comment (lahavyuv86) [reply deploy/clowdapp.yaml:46]: Fixed

Comment (theute) [reply deploy/clowdapp.yaml:46]: I really meant to use parameters instead of hard coded 500m and 500Mi values

File: deploy/clowdapp.yaml [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        readinessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /health/ready
+            port: 8000
+            scheme: HTTP
+          initialDelaySeconds: 40
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 1
```
Comment (theute) [inline deploy/clowdapp.yaml:38]: Port numbers still seem wrong

File: deploy/clowdapp.yaml (hunk 2 above)
Comment (lahavyuv86) [reply deploy/clowdapp.yaml:46]: Fixed

File: deploy/clowdapp.yaml (hunk 4 above)
Comment (lahavyuv86) [reply deploy/clowdapp.yaml]: Fixed

**Pull request merged**